bndtools.api/src/org/bndtools/api/central/ICentral.java

```diff
@@ -0,0 +1,22 @@
+package org.bndtools.api.central;
+
+import org.bndtools.api.ModelListener;
+import org.eclipse.jdt.core.IJavaProject;
+import org.osgi.annotation.versioning.ProviderType;
+
+import aQute.bnd.build.Project;
+
+@ProviderType
+public interface ICentral {
+
+	Project getModel(IJavaProject project);
+
+	void changed(Project model);
+
+	void addModelListener(ModelListener m);
+
+	void removeModelListener(ModelListener m);
+
+	void close();
```

Comment:
Is it really whise to have close on a service interface?

Reply:
Yes, you're probably right. In fact, I just did some code searches and found that the original `Central.close()` is not even called anywhere from within Bndtools, so I think we can safely remove it.